clarity_epp/export/illumina.py

```diff
@@ -148,28 +152,31 @@ def get_project(projects, urgent=False):
     # Urgent families / samples, skip deviating
     for family in [family for family in families.values() if family['urgent'] and not family['deviating']]:
         family_project = get_project(project_types[family['project_type']]['projects'], urgent=True)
-        for sample in family['samples']:
-            sample_sequence_name = get_sequence_name(sample)
-            sample_sequence_names[sample.name] = sample_sequence_name
+        for sample_artifact in family['samples']:
+            sample_sequence_name = get_sequence_name(sample_artifact)
+            for sample in sample_artifact.samples:
+                sample_sequence_names[sample.name] = sample_sequence_name
             sample_projects[sample_sequence_name] = family_project
             project_types[family['project_type']]['projects'][family_project] += 1
 
     # Deviating families / samples
     for family in [family for family in families.values() if family['deviating']]:
         family_project = get_project(project_types[family['project_type']]['projects'])
-        for sample in family['samples']:
-            sample_sequence_name = get_sequence_name(sample)
-            sample_sequence_names[sample.name] = sample_sequence_name
+        for sample_artifact in family['samples']:
+            sample_sequence_name = get_sequence_name(sample_artifact)
+            for sample in sample_artifact.samples:
+                sample_sequence_names[sample.name] = sample_sequence_name
```

Comment:
These 4 lines are repeated in line 155-158, 165-168, 176-179. The difference is which family is passed to the for loop, 
`family['urgent'] and not family['deviating']`
`not family['urgent'] and not family['deviating']`
`not family['urgent'] and not family['deviating']` 

Is it possible to avoid the code duplication? Might be able to merge the three code blocks.